Gruntfile.js

```diff
@@ -36,40 +36,23 @@ module.exports = (grunt) => {
         libraryExport: 'default',
       },
       resolve: {
-        extensions: ['.ts', '.js', '.json']
+        extensions: ['.ts', '.js', '.json'],
```

Comment:
comma is not needed

Reply:
I remove it to reduce changes, however all the other instances in the file have the comma.